core/src/cats/sql_get.cc

```diff
@@ -1078,6 +1078,23 @@ bool BareosDb::GetQueryDbids(JobControlRecord* jcr,
 }
 
 
+/**
+ * Get all volumes names
+ *
+ * Returns: false: on failure
+ *          true:  on success
+ */
+bool BareosDb::GetAllVolumeNames(db_list_ctx* volumenames)
+{
+  PoolMem query(PM_MESSAGE);
+  volumenames->clear();
+  Mmsg(query,
+       "SELECT DISTINCT Media.VolumeName FROM Media ORDER BY VolumeName");
+  SqlQueryWithHandler(query.c_str(), DbListHandler, volumenames);
+
+  return true;
```

Comment:
if you never fail, why return a value a all?

Reply:
actually forgot to run a check there, thanks!